backend.py

```diff
@@ -17,13 +17,16 @@ def __repr__(self):
 
 
 class Robot:
-    def __init__(self, direction, path, coordinates):
+    def __init__(self, direction, path, coordinates, lifecount, flagcount, injurycount):
```

Comment:
```suggestion
def __init__(self, direction, path, coordinates):
        self.direction = direction
        self.path = path
        self.coordinates = coordinates
        self.lifecount = 3
        self.flagcount = 0
        self.injurycount = 0
```
Nestačilo by jenom tohle? Pak už nebudeš muset používat funkci `set_new_robot` a ani ty parametry nebude potřeba zadávat, když robota vytváříš.

Reply:
Čím víc nad tím přemýšlím, tak si nejsem jistá, jestli to nevidím moc jednoduše...

Reply:
@JureckovaK  no nějak jsem to tam zabudovala, nechci to napevno dávat úplně do robota, protože by se brzo asi měly objevit metody typu "uber život", "přidej zranění", "seber vlajku" a čísla se pak budou dynamicky měnit.
Nejsem si ale jistá, jestli se to z mojí funkce opravdu dostává až do robota

Reply:
Ta moje myšlenka bude dávat smysl, pokud budeme roboty inicializovat ve hře jenom jedenkrát. Tzn. při vytváření nového (ne startovního) stavu hry se roboti nakopírují z původního stavu a upraví se jim životy/zranění/vlajky podle událostí herního kola.